index.html

```diff
@@ -7,24 +7,76 @@
     <title>Hair Salon Checkpoint</title>
     <link rel="stylesheet" href="./reset.css">
     <link rel="stylesheet" href="./style.css">
+    <link rel = "stylesheet" href = "./nav.css">
```

Comment:
no space for the attributes. you want to match line 8 and 9. where the equals sign is touching the attribute name and value. javascript allows spaces. you can get away with it in html but it's not the norm.